vespa-http-client/src/main/java/com/yahoo/vespa/http/client/config/SessionParams.java

```diff
@@ -8,12 +8,12 @@
 import java.util.List;
 
 /**
- * Parameters given to a {@link com.yahoo.vespa.http.client.SessionFactory}
- * when creating {@link com.yahoo.vespa.http.client.Session}s. This class is immutable
+ * Parameters given to a {@link com.yahoo.vespa.http.client.FeedClientFactory}
+ * when creating {@link com.yahoo.vespa.http.client.FeedClient}s. This class is immutable
  * and has no public constructor - to instantiate one, use a {@link Builder}.
  *
  * @author Einar M R Rosenvinge
- * @see com.yahoo.vespa.http.client.SessionFactory
+ * @see com.yahoo.vespa.http.client.FeedClientFactory
  * @see Builder
  */
 public final class SessionParams {
```

Comment:
Should we introduce `FeedClientParams` (or similar) to supersede this?

Reply:
I think it's ok as it is .. there are sessions against a node internally and these are parameters relating to that.